src/main/java/net/sf/jabref/logic/formatter/Formatters.java

```diff
@@ -45,7 +45,7 @@
 public class Formatters {
 
     public static final List<Formatter> CONVERTERS = Arrays.asList(
-            new HtmlToLatexFormatter(),
+            new HtmlToLatexFormatter(false), // TODO: Get the correct value from USE_CONVERT_TO_EQUATION
```

Comment:
I think we need a general strategy for how preferences in these formatters are handled. The idea of save actions was that these are specified in the bib-file and can be invoked on every save without destroying the bibitems. For this it is really necessary that every user runs the same save actions for the same file. But this is might be no longer the case if some formatters depend on user preferences. For the moment I would not change them but document which formatter depends on which preferences.